src/getting_started.md

```diff
@@ -62,28 +65,19 @@ on app.start {
 }
 ```
 
-Now run:
+Now compile the code to produce an AGC binary and then run it which prints `Hello, World!`:
 
 ```bash
 alan compile hello_world.ln hello_world.agc
-```
-
-This will produce an AGC binary that can be run like this:
-
-```bash
 alan run hello_world.agc
 ```
 
-which prints `Hello, World!`.
-
 You can also run the source file directly:
 
 ```bash
 alan hello_world.ln
 ```
 
-which will internally compile the code then run it, but you have to pay the compilation costs on each run.
```

Comment:
I don't know about removing this line.

Reply:
It is not in the README https://github.com/alantech/alan#recommended-usage

Reply:
I will add it back until the compiler is faster in both places

Reply:
That doesn't justify it's removal here. :)

Right now, the user may be wondering "why should I ever do the `alan compile ...` `alan run ...` dance?"

Reply:
It should be consistent.

Reply:
agreed